Modules/Server/Sources/PocketCastsServer/Public/Search/PodcastSearchTask.swift

```diff
@@ -66,18 +79,53 @@ public class PodcastSearchTask {
     public init(session: URLSession = .shared) {
         self.session = session
     }
+
     public func search(term: String) async throws -> [PodcastFolderSearchResult] {
+        var envelope: PodcastsSearchEnvelope = .failedResult
```

Comment:
Do we need that? Can't this just be optional?

This way we can remove the added static stuff.

Reply:
I was trying to have the same results as the old operation in order to be able to replace the operation code later on.

Reply:
I update the code to use nil instead of the empty response.